apps/backend/src/data/models/create-modifier-category-entity.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Column,
+  Entity,
+  ManyToOne,
+  OneToMany,
+  PrimaryGeneratedColumn
+} from 'typeorm';
+
+import { Product } from './create-product-entity';
+import { Modifiers } from './create-modifiers-entity';
+
+@Entity()
+export class ModifierCategory {
+  @PrimaryGeneratedColumn({ type: 'int' })
+  id: number;
+
+  @Column({ type: 'varchar', length: 255 })
+  name: string;
+
+  @Column({ type: 'int' })
+  minSelection: number;
+
+  @Column({ type: 'int' })
+  maxSelection: number;
+
+  @Column({ type: 'int' })
+  productId: number;
```

Comment:
you need to map to the correct field in the database. In your database you have `product_id` 
node_modules/typeorm/decorator/options/ColumnOptions.d.ts

Reply:
Fixed